vllm/v1/core/kv_cache_utils.py

```diff
@@ -621,6 +621,8 @@ def _get_kv_cache_config_uniform_type(vllm_config: VllmConfig,
     assert len(page_sizes) == 1
     page_size = page_sizes.pop()
 
+    # with cross-layer KV sharing, len(kv_cache_spec) may be less than no. of
+    # attention layers, in which case more KV cache blocks can be allocated
```

Comment:
People reading this part of code do not be aware of kv sharing. I think it's better to explain the implementation of kv sharing here.
```
if (kv_tgt_layer :=
                     attn_module.kv_sharing_target_layer_name) is not None:
                 validate_kv_target_layer(layer_name, kv_tgt_layer, layers)
                 # KV cache is shared with earlier layer, don't create a spec
                 self.shared_kv_cache_layers[layer_name] = kv_tgt_layer
                 continue
```